UnitedIncome/components
# [APP-5295] Wrap `CompletionScreen` in a container element

### Description

There is an interaction with browser zoom and `display: flex` and
explicit width on the `uic--mcg-completion-screen` that causes Google
Chrome to display the contents of the `CompletionScreen` component
within a `McGonagall` flow offset. This issue was encounted only at
browser viewport width greater than 1483px (on a retina screen) with
browser zoom set to less than 100%.

By wrapping `uic--mcg-completion-screen` in a flexbox container with
explicit width the issue no longer occurs.

### Testing Instructions

I have not found a way to test this without the full application. It will require usage in https://github.com/UnitedIncome/client to test. I think the issue occurs in more places than this specific flow but this is where I tested it.

1. Using Google Chrome, set browser zoom to 90%
1. Log in as an account with the `MultipleAccountCreator` role
1. Navigate to advisor tools
1. Navigate to "Open Multiple New Member Accounts"
1. Go through the flow with a valid CSV file
1. Note that the layout is centered

### Screenshots

**Before**

<img width="1595" alt="Screen Shot 2020-06-10 at 4 17 14 PM" src="https://user-images.githubusercontent.com/65303564/84314512-31c81000-ab36-11ea-99e8-98151521f8e3.png">

**After**

<img width="1595" alt="Screen Shot 2020-06-10 at 4 14 53 PM" src="https://user-images.githubusercontent.com/65303564/84314525-37bdf100-ab36-11ea-99f0-85768f159d35.png">

### Comments

I don't understand the browser interaction causing this to happen, it is
only occurring in Google Chrome and I suspect this is a browser bug and
that this change _should_ not be necessary.

Refs [APP-5295](https://unitedincome.atlassian.net/browse/APP-5295)
